src/test/server/server_test_runner.cpp

```diff
@@ -125,4 +126,33 @@ TEST_F(ServerTestRunner, TestPreparedStatement) {
   EXPECT_EQ(result2.size(), 2u);
 }
 
+TEST_F(ServerTestRunner, TestParallelConnections) {
```

Comment:
This is in the hyriseSystemTest and not in the hyriseTest, right? If it is, that's cool, otherwise the time this could take would potentially concern me

Reply:
in systemTest for exactly that reason